### ISSUE TITLE
Default card still not being first

### ISSUE THREAD

**Issue by cezeta** [Muirey03/Amandus#10]
Hello,
I have iOS 13.5 with the new unc0ver jailbreak... the tweak still does not display the default card as first one - please take a look at it (I tried changing the default card to see if it will fix itself, but it did not help)

**Comment by ZeldaZach** [comment]
Agreed with the original poster. Would love to see the tweak remember the preferred card (or the most recently used card) for quicker operations. Great tweak, thanks for the contribution to the community!

**Comment by systox** [comment]
This is still an issue. The last card in wallet is shown as the default card when initiating a purchase with tweak installed. It should instead show the default selected payment method.

This causes a problem because when paying online the tweak is bypassed and the default card is used. So you can’t just set the card you want to use universally as the last card in wallet app.
